frontend/src/app/components/work-packages/wp-relations-count/wp-watchers-count.component.ts

```diff
@@ -2,7 +2,6 @@ import {Component, Input, OnDestroy, OnInit} from '@angular/core';
 import {componentDestroyed} from 'ng2-rx-componentdestroyed';
 import {takeUntil} from 'rxjs/operators';
 import {WorkPackageCacheService} from '../../work-packages/work-package-cache.service';
-import {combineLatest} from 'rxjs';
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/opf/openproject/blob/b6e4680ef80bc97abdec363477c560842ceb1115/frontend/src/app/components/work-packages/wp-relations-count/wp-watchers-count.component.ts#L5"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</td>
    </tr>
  </tbody>
</table>